src/plugins/intel_cpu/src/nodes/interpolate.cpp

```diff
@@ -3281,11 +3289,13 @@ size_t Interpolate::getSpatialDimsNum(const Dim rank) {
 }
 
 bool Interpolate::canFuse(const NodePtr& node) const {
-    const bool is_i8_nearest_planar = interpAttrs.mode == InterpolateMode::nearest &&
-               interpAttrs.layout == InterpolateLayoutType::planar &&
-               interpAttrs.inPrc == InferenceEngine::Precision::I8 &&
-               interpAttrs.outPrc == InferenceEngine::Precision::I8;
-    if (!mayiuse(cpu::x64::sse41) || is_i8_nearest_planar || interpAttrs.mode == InterpolateMode::linear) {
+    const bool isNearestPlanar = interpAttrs.mode == InterpolateMode::nearest &&
+                                 interpAttrs.layout == InterpolateLayoutType::planar;
+    const bool isI8OrU8 =
+        (interpAttrs.inPrc == InferenceEngine::Precision::I8 && interpAttrs.outPrc == InferenceEngine::Precision::I8) ||
+        (interpAttrs.inPrc == InferenceEngine::Precision::U8 && interpAttrs.outPrc == InferenceEngine::Precision::U8);
+
+    if (!mayiuse(cpu::x64::sse41) || (isNearestPlanar && isI8OrU8) || interpAttrs.mode == InterpolateMode::linear) {
```

Comment:
As far as I understand, interpAttrs.layout, interpAttrs.inPrc and interpAttrs.outPrc is not finalized yet at the time when this function is called. If jit kernel does not support fusion, here need an overkilled conditions, interpAttrs.mode == InterpolateMode::nearest  and getOriginalInputPrecisionAtPort(DATA_ID) == I8 or U8, no matter what layout it is. Ideally we should support fusion at all jitted kernels and remove these restriction.

Reply:
This code is removed as we decided to support fusing for `int8` later